tests/build/build.go

```diff
@@ -729,27 +653,13 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build-ser
 				timeout = time.Second * 10
 				interval = time.Second * 2
 				Consistently(func() error {
-					switch gitProvider {
-					case "github":
-						prs, err := f.AsKubeAdmin.CommonController.Github.ListPullRequests(helloWorldComponentGitSourceRepoName)
-						Expect(err).ShouldNot(HaveOccurred())
-
-						for _, pr := range prs {
-							if pr.Head.GetRef() == pacBranchName {
-								return fmt.Errorf("did not expect a new PR created in %s repository after initial PaC configuration was already merged for the same component name and a namespace", helloWorldComponentGitSourceRepoName)
-							}
-						}
-						return nil
-					case "gitlab":
-						mrs, err := f.AsKubeAdmin.CommonController.Gitlab.GetMergeRequests()
-						Expect(err).ShouldNot(HaveOccurred())
+					prs, err := gitClient.ListPullRequests(helloWorldRepository)
+					Expect(err).ShouldNot(HaveOccurred())
 
-						for _, mr := range mrs {
-							if mr.SourceBranch == pacBranchName {
-								return fmt.Errorf("did not expect a new PR created in %s repository after initial PaC configuration was already merged for the same component name and a namespace", helloWorldComponentGitSourceRepoName)
-							}
+					for _, pr := range prs {
+						if pr.SourceBranch == pacBranchName {
+							return fmt.Errorf("did not expect a new PR created in %s repository after initial PaC configuration was already merged for the same component name and a namespace", helloWorldComponentGitSourceRepoName)
```

Comment:
`s/helloWorldComponentGitSourceRepoName/helloWorldRepository/`

Reply:
`helloWorldRepository` has a different format when using GitHub (`repository_name` only) and GitLab (`organization/repository_name`). 
I chose to keep the behavior consistent and use the repository name only. 

Would you prefer to use the `org/repo_name` format for both providers?

Reply:
owner(user or org)/repo is more common, imo